addons/mail/static/src/web/thread_service_patch.js

```diff
@@ -164,6 +167,7 @@ patch(ThreadService.prototype, "mail/web", {
                       })
                     : false,
                 checked: true,
+                customerInfo: customerInfo,
```

Comment:
Object initializers support the definition of properties from just a reference to an identifier. The resulting property will have the identifier as its key and the corresponding value as its value.

In other words, here's a handy shorthand to avoid repeating yourself when the key and the value both correspond to the same identifier:
```suggestion
                customerInfo,
```